packages/bpk-component-image/src/_bpk-image.scss

```diff
@@ -0,0 +1,50 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins/index';
+
+@mixin bpk-image {
+  position: relative;
+  display: block;
+}
+
+@mixin bpk-image__content {
+  width: 100%;
+  height: 100%;
+  transition: opacity $bpk-duration-base ease-in-out;
+  opacity: 0;
+}
+
+@mixin bpk-image__content--show {
+  opacity: 1;
+}
+
+@mixin bpk-image__spinner {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  display: absolute;
+  align-items: center;
```

Comment:
This only does anything when you use `display: flex`. There's two ways you can center the spinner, either using flexbox or using absolute positioning. Pick one :)